cellpack/autopack/IOutils.py

```diff
@@ -567,180 +568,12 @@ def addCompartments(env, compdic, i, io_ingr):
-                        ingr.o_name
-                    ]["include"] = str(ingr.o_name + ".json")
-                else:
-                    env.jsondic["compartments"][str(o.name)]["surface"]["ingredients"][
-                        ingr.o_name
-                    ] = io_ingr.ingrJsonNode(
-                        ingr
-                    )  # {"name":ingr.o_name}
-                    env.jsondic["compartments"][str(o.name)]["surface"]["ingredients"][
-                        ingr.o_name
-                    ]["name"] = ingr.o_name
-        ri = o.innerRecipe
-        if ri:
-            env.jsondic["compartments"][str(o.name)]["interior"] = OrderedDict()
-            env.jsondic["compartments"][str(o.name)]["interior"][
-                "ingredients"
-            ] = OrderedDict()
-            for ingr in ri.ingredients:
-                if useXref:
-                    # write the json file for this ingredient
-                    io_ingr.write(
-                        ingr, pathout + os.sep + ingr.o_name, ingr_format="json"
-                    )
-                    # use reference file
-                    env.jsondic["compartments"][str(o.name)]["interior"]["ingredients"][
-                        ingr.o_name
-                    ] = OrderedDict()
-                    env.jsondic["compartments"][str(o.name)]["interior"]["ingredients"][
-                        ingr.o_name
-                    ]["name"] = ingr.o_name
-                    env.jsondic["compartments"][str(o.name)]["interior"]["ingredients"][
-                        ingr.o_name
-                    ]["include"] = str(ingr.o_name + ".json")
-                else:
-                    env.jsondic["compartments"][str(o.name)]["interior"]["ingredients"][
-                        ingr.o_name
-                    ] = io_ingr.ingrJsonNode(
-                        ingr
-                    )  # {"name":ingr.o_name}
-                    env.jsondic["compartments"][str(o.name)]["interior"]["ingredients"][
-                        ingr.o_name
-                    ]["name"] = ingr.o_name
-
-    def default(o):
-        raise TypeError(repr(o) + " is not JSON serializable ", o, type(o))
-
-    with open(setupfile, "w") as fp:  # doesnt work with symbol link ?
-        if indent:
-            json.dump(
-                env.jsondic, fp, indent=1, separators=(",", ":"), default=default
-            )  # ,indent=4, separators=(',', ': ')
-        else:
-            json.dump(
-                env.jsondic, fp, separators=(",", ":"), default=default
-            )  # ,indent=4, separators=(',', ': ')
-    print("recipe saved to ", setupfile)
+def save_as_simularium(env, setupfile, all_ingr_as_array):
+    autopack.helper.clear()
+    autopack.helper.init_scene_with_objects(all_ingr_as_array)
+    autopack.helper.writeToFile(None, f"{setupfile}_results", env.boundingBox)
+    webbrowser.open("https://simularium.allencell.org/viewer")

```

Comment:
Mentioned in the call, but might be nice to be able to just save to simularium directly, then another function to open it in the browser as well.